geniza/corpus/models.py

```diff
@@ -702,6 +702,12 @@ def index_data(self):
                     if f.manifest
                     for img in f.iiif_images()[0]
                 ],
+                "iiif_labels_ss": [
+                    label
+                    for f in fragments
+                    if f.manifest
+                    for label in f.iiif_images()[1]
```

Comment:
we should only call `iiif_images` once for efficiency; I think it will make the code more readable too...

Reply:
wonder how this is going to impact indexing time, actually... can/should we add prefetching on manifests and canvases? 😬

Reply:
here's my suggestion for revising:

```python
        images = self.iiif_images()  
        index_data.update(
              # ...
               "iiif_images_ss": [
                    img['image'].info()[:-10]  # i.e., remove /info.json
                    for img in images
                ],
                "iiif_labels_ss": [img['label'] for img in images]
             # ...
````

Reply:
Done in fa601a7, I think—unless we still need to add prefetching?

Reply:
let's try without for now